apps/angular/dialog-alert-form/src/app/pages/page.component.ts

```diff
@@ -0,0 +1,17 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { toSignal } from '@angular/core/rxjs-interop';
+import { ActivatedRoute } from '@angular/router';
+import { map } from 'rxjs';
+
+@Component({
+  standalone: true,
+  template: `
+    <section>
+      <h1>{{ title() }}</h1>
+    </section>
+  `,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PageComponent {
+  title = toSignal(inject(ActivatedRoute).data.pipe(map((d) => d['title'])));
```

Comment:
you can use route input to archive this in a prettier way 👍

Reply:
yes, good point